java/ci/README.md

```diff
@@ -11,32 +11,30 @@
 
 In the root path of cuDF repo, run below command to build the docker image.
 ```bash
-docker build -f java/ci/Dockerfile.centos7 --build-arg CUDA_VERSION=10.1 -t cudf-build:10.1-devel-centos7 .
+docker build -f java/ci/Dockerfile.centos7 --build-arg CUDA_VERSION=11.0 -t cudf-build:11.0-devel-centos7 .
 ```
 
 We support different CUDA versions as below:
```

Comment:
This is now worded a bit weirdly since there's only one version listed.  I suggest changing it to something like this:
```suggestion
The following CUDA versions are supported:
```

Reply:
thx! updated